### ISSUE TITLE
Some icons in Numix-Circle are not application icons

### ISSUE THREAD

**Issue by hugomg** [numixproject/circle-core#3337]
I cross-checked the applications icon list in Numix Circle with the freedesktop.org [Icon Naming Specification](https://specifications.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html) and identified a non-exhaustive list of Numix Circle icons that are not application icons.

I think these icons should be either be:
* Removed from Numix Circle, to avoid clashes; or
* Moved back to the base Numix theme; or
* Put in their own folders (actions, devices, etc) instead of the apps folder, to clarify the intent.

The icons I identified are in the following tables, organized according to the Icon Naming Specification:

## Standard Action Icons

| Image | Icon Name |
| --- | :--- |
| ![system-lock-screen](https://cloud.githubusercontent.com/assets/1452192/17278866/9e2e8c8e-573d-11e6-91bb-2a6423980939.png) | system-lock-screen |
|![system-search](https://cloud.githubusercontent.com/assets/1452192/17278877/b719e39c-573d-11e6-9e39-0aaa0419df50.png) | system-search |

## Standard Category Icons

| Image | Icon Name |
| --- | :--- |
| ![applications-accessories](https://cloud.githubusercontent.com/assets/1452192/17278889/f6a9dbc0-573d-11e6-9f40-7a30da4fc6da.png) | applications-accessories |
| ![applications-development](https://cloud.githubusercontent.com/assets/1452192/17278890/fdbd23b8-573d-11e6-8d06-cb73250360f4.png) | applications-development |
| ![applications-games](https://cloud.githubusercontent.com/assets/1452192/17278892/085d4bae-573e-11e6-8a74-1d9ddb5d4da8.png) | applications-games |
| ![applications-graphics](https://cloud.githubusercontent.com/assets/1452192/17278894/0feebefc-573e-11e6-9164-c4942e464de0.png) | applications-graphics |
| ![applications-internet](https://cloud.githubusercontent.com/assets/1452192/17278897/1db02148-573e-11e6-98c7-ddd15895e350.png) | applications-internet |
| ![applications-multimedia](https://cloud.githubusercontent.com/assets/1452192/17278900/259185be-573e-11e6-9acd-62ba758126c9.png) | applications-multimedia |
| ![applications-office](https://cloud.githubusercontent.com/assets/1452192/17278902/2baa76d6-573e-11e6-8b3b-06ee3a667b5c.png)  | applications-office |
| ![applications-other](https://cloud.githubusercontent.com/assets/1452192/17278906/30b63caa-573e-11e6-814f-27242238c54f.png) |  applications-other |
| ![applications-science](https://cloud.githubusercontent.com/assets/1452192/17278909/3e626806-573e-11e6-8d71-9f292df0e754.png) | applications-science |
| ![applications-system](https://cloud.githubusercontent.com/assets/1452192/17278910/45b14b7c-573e-11e6-9b89-eda3a0729485.png) | applications-system |
|  ![preferences-desktop](https://cloud.githubusercontent.com/assets/1452192/17278914/4e73118c-573e-11e6-9837-41a0fcaeddd6.png) | preferences-desktop |
| ![preferences-desktop-peripherals](https://cloud.githubusercontent.com/assets/1452192/17278915/55d28d0e-573e-11e6-8938-962e6b40c529.png) | preferences-desktop-peripherals |
| ![preferences-system](https://cloud.githubusercontent.com/assets/1452192/17278919/6e322ecc-573e-11e6-8a6e-20b1c9c6b6fc.png) | preferences-system |
| ![preferences-system-network](https://cloud.githubusercontent.com/assets/1452192/17278922/752032ba-573e-11e6-8d4a-54166ff3f6db.png) | preferences-system-network |
| ![system-help](https://cloud.githubusercontent.com/assets/1452192/17278924/7c05b4e2-573e-11e6-883a-171c1d4030f7.png) | system-help |

## Standard Device Icons

| Image | Icon Name |
| --- | :--- |
| ![battery](https://cloud.githubusercontent.com/assets/1452192/17278935/c5f1179a-573e-11e6-9d9c-c9b1c7929871.png)| battery|
| ![input-keyboard](https://cloud.githubusercontent.com/assets/1452192/17278936/c9b86c84-573e-11e6-95c1-daa7c0e451be.png) | input-keyboard|
|![input-mouse](https://cloud.githubusercontent.com/assets/1452192/17278937/ce137332-573e-11e6-8a1f-4b019130b3a9.png) | input-mouse|
|![input-tablet](https://cloud.githubusercontent.com/assets/1452192/17278938/d1a758ec-573e-11e6-800d-268b479be2a6.png) | input-tablet|
|![printer](https://cloud.githubusercontent.com/assets/1452192/17278939/d6327a4a-573e-11e6-80fb-31ae0fcbf31e.png) | printer|
|![scanner](https://cloud.githubusercontent.com/assets/1452192/17278940/d91ca49c-573e-11e6-895a-c6db18bfe3a2.png) | scanner|

# 10. Standard MIME Type Icons

| Image | Icon Name |
| --- | :--- |
| ![x-office-address-book](https://cloud.githubusercontent.com/assets/1452192/17278947/e6af8278-573e-11e6-95df-0af8f8a77767.png) | x-office-address-book |
| ![x-office-calendar](https://cloud.githubusercontent.com/assets/1452192/17278948/ea26e77a-573e-11e6-9348-10b255084a0c.png) | x-office-calendar |

# 11. Standard Place Icons

| Image | Icon Name |
| --- | :--- |
| ![network-workgroup](https://cloud.githubusercontent.com/assets/1452192/17278949/f3777e0c-573e-11e6-838d-7f70892fee98.png) | network-workgroup |

# 12. Standard Status Icons

| Image | Icon Name |
| --- | :--- |
| ![security-medium](https://cloud.githubusercontent.com/assets/1452192/17278954/2d15ffda-573f-11e6-9e70-c516c95bbeeb.png) | security-medium|

Note: security-high and security-low, which are present in the base Numix theme, are missing. Maybe this icon was an accidental name clash?

**Comment by palob** [comment]
We are well aware of that. However regrettably there are applications which use icons named that way for  ``.desktop`` launchers. 

Actually it's individual issues to be reported against the respective upstream bug trackers.

(``applications-...`` are for the Cinnamon menu, ``preferences-...`` for control centers of some DEs).

It's a trade-off between Numix Circle icons showing up in places they shouldn't or differently designed icons amongst Numix Circle icons.
For the sake of design consistency I think the first case is less crirical.

So before we can remove those icons from Numix Circle we need to be sure that there isn't any app out there using it. Even if some app got fixed upstream we'd probably have to wait some time in order not to break it with LTS distros.

**Comment by Foggalong** [comment]
Generally sorting application icons into folders based on where they're used is something we might have to look at at some point. I've seen themes use additional folders like `games`, `web`, `distros`, etc to make finding stuff easier.

Regarding this issue though and just removing them from circle isn't something we can do because upstream developers have already made the bad choices of icon names.

**Comment by Foggalong** [comment]
Gonna mark this as a wontfix for now for the reasons stated above by myself and @palob. However, there's an issue open (numixproject/numix-core#5) for sorting icons into categories by usage.
